lib/phoenix/digester.ex

```diff
@@ -20,8 +20,8 @@ defmodule Phoenix.Digester do
     * `input_path` - The path where the assets are located
     * `output_path` - The path where the compiled/compressed files will be saved
   """
-  @spec compile(String.t, String.t) :: :ok | {:error, :invalid_path}
-  def compile(input_path, output_path) do
+  @spec compile(String.t, String.t, Bool.t) :: :ok | {:error, :invalid_path}
+  def compile(input_path, output_path, version_param) do
```

Comment:
We can use `version_param?` in all the places to improve readability and highlight variable nature.